test/agentchat/contrib/test_gpt_assistant.py

```diff
@@ -24,7 +24,7 @@
 
 if not skip:
     config_list = autogen.config_list_from_json(
-        OAI_CONFIG_LIST, file_location=KEY_LOC, filter_dict={"api_type": ["openai"]}
+        OAI_CONFIG_LIST, file_location=KEY_LOC, filter_dict={"api_type": ["openai", "azure"]}
     )
```

Comment:
Keep the original config_list as openai_config_list
Add an aoai_config_list with filter_dict={"api_version": ["2024-02-15-preview"]}
Then test these two config_lists separately